flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolTestBase.java

```diff
@@ -0,0 +1,72 @@
+package org.apache.flink.runtime.jobmaster.slotpool;
```

Comment:
The license is missing and flink will not compile. https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=5111&view=logs&j=52b61abe-a3cc-5bde-cc35-1bbe89bb7df5&t=54421a62-0c80-5aad-3319-094ff69180bb.